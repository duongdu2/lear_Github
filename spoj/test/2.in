3
9
31 20 70 80 12 38 78 78 47 
8
48 34 15 96 66 61 94 5 
8
91 34 79 56 24 22 48 25 
