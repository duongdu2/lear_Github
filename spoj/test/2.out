236
256
226
